resources/js/components/small/AdminShowMessage.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <div v-if="message" class="break_work">
```

Comment:
I think you mean 'break-word'. I first typed that wrong as well just now. It's so that if someone messages facesmash words with no spaces, it doesn't stretch the entire modal. It's because I don't trust people.

Reply:
fixed